pkg/objectcache/k8scache/k8scache.go

```diff
@@ -94,8 +96,13 @@ func (k *K8sObjectCacheImpl) DeleteHandler(_ context.Context, obj *unstructured.
 		if err != nil {
 			return
 		}
-		k.podSpec.Delete(podKey(pod.GetNamespace(), pod.GetName()))
-		k.podStatus.Delete(podKey(pod.GetNamespace(), pod.GetName()))
+
+		// delete the pod spec and status after 1 minute
```

Comment:
Why?

Reply:
We talked about this... to have the pod status available in case the pod is deleted...